circles-app/.env.example

```
@@ -4,4 +4,5 @@ DB_USER =
 DB_DATABASE =
 DB_HOST =
 DB_PORT =
-DB_SSL_MODE =
\ No newline at end of file
+DB_SSL_MODE =
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)